pycheribuild/projects/cross/apache.py

```diff
@@ -129,3 +129,36 @@ def configure(self, **kwargs):
                          cwd=self.build_dir / "server")
             self.run_cmd(str(self.host_CC), "gen_test_char.lo", "-o",
                          "gen_test_char", cwd=self.build_dir / "server")
+
+class BuildSSLProc(CrossCompileCMakeProject):
+    target = "sslproc"
+
+    repository = GitRepository("https://github.com/CTSRD-CHERI/sslproc.git")
+
+    native_install_dir = DefaultInstallDir.IN_BUILD_DIRECTORY
+    cross_install_dir = DefaultInstallDir.ROOTFS_LOCALBASE
```

Comment:
This should be the default?

Reply:
For both or just cross_install_dir?  (I had read this as only being for cross_install_dir at first and only removed that one, but on re-reading I see you might have meant both?)

Reply:
I don't recall what I meant, but it does apply to both